src/lib/jsonParseV2/parseContextAttribute.cpp

```diff
@@ -258,7 +258,7 @@ std::string parseContextAttribute(ConnectionInfo* ciP, ContextAttribute* caP)
 
   if (document.HasParseError())
   {
-    OrionError oe(SccBadRequest, "Errors found in incoming JSON buffer");
+    OrionError oe(ERROR_STRING_PARSERROR, "Errors found in incoming JSON buffer");
```

Comment:
I have a global search to use "ParserError" in all cases where "Errors found in incoming JSON buffer" are reported (as it has to be, according to the error rules defined at https://github.com/telefonicaid/fiware-orion/issues/1286)